ext/opcache/tests/func_info.phpt

```diff
@@ -0,0 +1,85 @@
+--TEST--
+Test that return types in zend_func_info.c match return types in stubs
+--INI--
+opcache.enable=1
+opcache.enable_cli=1
+opcache.optimization_level=-1
+--SKIPIF--
+<?php require_once('skipif.inc'); ?>
+--FILE--
+<?php
+
+$extensions = [
+    "standard",
+    "date",
+    "pcre",
+    "mysqli",
+    "curl",
+    "mbstring",
+    "iconv",
+    "json",
+    "xml",
+    "zlib",
+    "hash",
+    "sodium",
+    "session",
+    "pgsql",
+    "bcmath",
+    "exif",
+    "filter",
+    "gettext",
+    "gd",
+    "spl",
+];
+
+$contents = "<?php\n";
+
+foreach ($extensions as $extension) {
+    try {
+        $reflectionExtension = new ReflectionExtension($extension);
+
+        foreach ($reflectionExtension->getFunctions() as $function) {
+            $contents .= "try {\n";
+            $contents .= "    {$function->getName()}(null, null, null, null, null, null, null, null);\n";
+            $contents .= "} catch (ArgumentCountError|Error) {\n";
```

Comment:
I don't think you can use try/catch. You need to generate everything inside a function (and you don't even need to call it).